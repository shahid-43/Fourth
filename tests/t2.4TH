10 2 / . CR .
5 