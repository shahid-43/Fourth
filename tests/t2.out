5

Stack content: [Integer 5]